### PR TITLE
Fix issue-1636, create migration guide for moving models to new repos…

### PR DESCRIPTION
**Pull request by somesh-kumar**
…itory

Signed-off-by: Somesh Kumar <somesh.kumar@bosch-si.com>

### CODE REVIEW

**Comment by aedelmann** [comment]
Build fails because there a more broken links ins your documentation. Please refer to build log for details

**Comment by somesh-kumar** [comment]
Fixed dead link for image.

File: docs/tutorials/migrate_vorto_models.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	![download model](../images/tutorials/migrate_model/download_model.png)
+
+4. Log in to new [Vorto Repository](https://vorto.eclipse.org)
+5. Click **Import** and select the zip file you downloaded in Step-3
+6. Now select your desired namespace from the dropdown (your model will be imported into this namespace)
+7. Click **Upload** 
+8. If all goes well you will receive a success message *model is valid and ready for import*
+9. Click **Import** to finish uploading
+
+## Explore
```
Comment (aedelmann) [inline docs/tutorials/migrate_vorto_models.md]: Rename to What's next ? 
Bullet points to other interesting , related topics, such as Manage collaborators, Create New Model , ..

File: docs/tutorials/migrate_vorto_models.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Migrating your Vorto models to the new Vorto Repository
+
+----------
+This guide will walk you through the process of migrating your existing Vorto models into the new Vorto repository. Let's get started.
+
+## Prerequisite
+- Login to the new [Vorto Repository](https://vorto.eclipse.org) and [Setup your Repository and Namespace](./create_repository.md)
```
Comment (aedelmann) [inline docs/tutorials/migrate_vorto_models.md]: - Github ID
- Existing Vorto Models in the old repository

**Comment by aedelmann** [comment]
Hi @somesh-kumar 
Is this now updated from your side, so I can review it or does it still need to undergo changes?

**Comment by somesh-kumar** [comment]
@aedelmann , please review

File: docs/tutorials/migrate_vorto_models.md (hunk 1 above)
Comment (somesh-kumar) [reply docs/tutorials/migrate_vorto_models.md]: done

**Pull request merged**